app/controllers/rounds_controller.rb

```diff
@@ -40,9 +34,70 @@ def update
     end
   end
 
+  def activate
+    if @round.valid?
+      if @round.active == false
```

Comment:
Whoops, one more boolean comparator.
```suggestion
      if !@round.active
```